app/assets/stylesheets/app/tables.scss

```diff
@@ -24,3 +24,15 @@ table {
 tfoot.cart {
   border-top: 3px solid $navbarActiveBorderColor;
 }
+
+@media screen and (min-width: 980px) {
```

Comment:
I think I’d like to leave these separate for now, because ideally the width is independent and based on the components which look different at those different widths, and `responsive.css` is geared towards the navigation system currently.

I picked 980px as a starting point here which ended up working pretty well, but I think we should be able to tweak this value depending on how the table grows and shrinks, and independently of other breakpoints that may work better for other content.